docs/docs/namespaces.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Flora Namespaces
+slug: namespaces
+---
+
+In Flora, packages are categorised in a namespace to mark their provenance. They start with a `@` and allow us to refer to packages unambiguously or mark their importance.
+
+## @haskell Packages
+
+Some packages are foundational to the ecosystem and maintained by either the Core Libraries Committee or the GHC team, and this makes them unique in terms of the expectations we have from them.
+
+These packages live in the [`@haskell`] namespace to show that they are stable and reliable. Some example of packages are `base`, `text`, `bytestring`, and `mtl`.
+
+## @hackage Packages
+
+This is where most third-party packages live, which you will find on [Hackage](https://hackage.haskell.org).
+
+## @cardano Packages
```

Comment:
Is it worth putting a brief word on (or a link to) the Cardano index? I've heard of CHaP but had to look up what its purpose was.

Reply:
The link in the first sentence doesn't work for you?

Reply:
Doh, hadn't rendered and didn't realise you could do links via footnotes in markdown.

Reply:
oh yeah it's very useful. :)